docs/content/docs/connectors/table/hive/hive_read_write.md

```diff
@@ -382,6 +382,38 @@ SELECT * FROM hive_table WHERE dt='2020-05-20' and hr='12';
 
 ```
 
+```sql
```

Comment:
Please add a short description (what's the purpose) for the following example. The **Full Example** is not just for your new introduced config option, most users don't have the context.